packages/connectors/connector-aws-ses/src/index.test.ts

```diff
@@ -64,4 +64,9 @@ describe('sendMessage()', () => {
       })
     ).rejects.toThrow();
   });
+
+  /**
+   * TODO: Add real test cases here to ensure the connector works as expected.
+   * For example, a real send email request triggers `crypto.getRandomValues` not supported error.
+   */
 });
```

Comment:
we can remove this then. it should come with the connector integration tests in the future